core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationRecord.java

```diff
@@ -33,6 +35,9 @@
   /** Set that contains the addresses of lost nodes. */
   private final Set<Address> mLostNodes;
 
+  /** Listeners to call when needs to check configuration. */
+  private final BlockingQueue<Runnable> mRegenerateReportListeners = new LinkedBlockingQueue<>();
```

Comment:
why `BlockingQueue`/`LinkedBlockingQueue`?

Reply:
Changed to List, thanks